# TASK 0: ARTICLE ON TRANSCRIPTOMICS


## **Differential Expression: Downregulation and Upregulation**

Differential expression plays a crucial role in cancer development and progression. Cancer cells often exhibit aberrant gene expression patterns compared to normal cells. Upregulation of oncogenes, genes that promote cell growth and division, can drive uncontrolled cell proliferation and tumor formation. Conversely, downregulation of tumor suppressor genes, which normally inhibit cell growth, can contribute to cancer development. By studying differential expression patterns in cancer cells, researchers can identify potential therapeutic targets and develop novel treatments.

The primary challenge to realizing the potential of high-throughput technologies is gaining biological insight from the generated data (Maleki et al., 2020). Over the last few decades, bioinformatics methods have been developed and improved on. More studies have focused on the signaling and metabolic pathways of cancer diseases, data mining and validation of related biomolecular targets (Wu and Xi, 2021). Differential gene expression is a prominent approach to interpreting gene expression data. It identifies upregulation or depletion of expression levels in  defined genes  associated with disease pathways or conditions (Maleki et al., 2020). Notably, Gene Set Analysis (GSA) has been commonly used in recent years and also improved upon. GSA methods include Hypergeometric test, Fisher's exact test, Gene Set Enrichment Analysis (GSEA) and many more.

Recent research studies have employed GSA to comprehend the anomalous molecular mechanisms linked to cancer diseases(Guo et al., 2017; Tan et al., 2018; Park et al., 2023). For instance, Guo et al. (2017) utilized GSEA to explore CKAP2-associated pathways in cervical carcinoma, identifying the FAK signaling pathway prominently linked to heightened CKAP2 expression and other metastasis-related and cell cycle pathways. Other studies adopted GSA knowledgebase-derived gene sets to discern enriched differentially expressed genes pertinent to HPV-induced carcinogenesis and cervical cancer progression (Gan et al., 2023). Furthermore, the GSA approach has been used to facilitate the recognition of transcriptomic changes between treated and untreated cervical cancer cohorts (Deng et al., 2023).

Differential expression is potent for biomarker discovery  especially in cancer research. It identifies genes differentially expressed between cancer and normal tissues, aiding early detection, prognosis, and personalized treatment. With technological advancements,  its use is posed to be very effective.

 
## **REFERENCES**

Deng, Y., Song, Y., Du, Q., Wang, C. C., Li, H., Sui, Y., Zhang, Y., & Tang, T. (2023). Anti-HPV16 oncoproteins siRNA therapy for cervical cancer using a novel transdermal peptide PKU12. Frontiers in Oncology , 13 , 1175958. <https://doi.org/10.3389/fonc.2023.1175958>

Gan, Q., Mao, L., Shi, R., Chang, L., Wang, G., Cheng, J., & Chen, R. (2023). Prognostic Value and Immune Infiltration of HPV-Related Genes in the Immune Microenvironment of Cervical Squamous Cell Carcinoma and Endocervical Adenocarcinoma. Cancers , 15 (5), 1419. <https://doi.org/10.3390/cancers15051419>

Guo, Q., Song, Y., Hua, K., & Gao, S. (2017). Involvement of FAK-ERK2 signaling pathway in CKAP2-induced proliferation and motility in cervical carcinoma cell lines. Scientific Reports , 7 , 2117. <https://doi.org/10.1038/s41598-017-01832-y>

Maleki, F., Ovens, K., Hogan, D. J., & Kusalik, A. J. (2020). Gene Set Analysis: Challenges, Opportunities, and Future Research. Frontiers in Genetics , 11. <https://www.frontiersin.org/articles/10.3389/fgene.2020.00654>

Park, N. J.-Y., Choi, Y., Lee, D., Park, J. Y., Kim, J. M., Lee, Y. H., Hong, D. G., Chong, G. O., & Han, H. S. (2023). Transcriptomic Network Analysis Using Exfoliative Cervical Cells Could Discriminate a Potential Risk of Progression to Cancer in HPV-related Cervical Lesions: A Pilot Study. Cancer Genomics & Proteomics , 20 (1), 75–87. <https://doi.org/10.21873/cgp.20366>

Tan, M. S., Chang, S.-W., Cheah, P. L., & Yap, H. J. (2018). Integrative machine learning analysis of multiple gene expression profiles in cervical cancer. PeerJ , 6 , e5285. <https://doi.org/10.7717/peerj.5285>

Wu, B., & Xi, S. (2021). Bioinformatics analysis of differentially expressed genes and pathways in the development of cervical cancer. BMC Cancer , 21 (1), 733. <https://doi.org/10.1186/s12885-021-08412-4>

 
